app/views/shared/_progress_navigation_buttons.html.erb

```diff
@@ -20,7 +20,7 @@
     <% if confirmation_link %>
       <%= h(link_to l10n("confirm"), checkout_insured_plan_shopping_path(@enrollment.id, plan_id: @plan.id, market_kind: @market_kind, coverage_kind: @coverage_kind), class: "btn btn-primary btn-block disabled", id: 'btn-continue', method: :post, disabled: !@enrollable) %>
     <% elsif !button_type.nil? %>
-      <button id="btn-continue" <%= "disabled=#{disable_link}" if disable_link == true %> type="<%= button_type %>" method="<%= link_method %>">
+      <button id="btn-continue" <%= "disabled=#{disable_link}" %> type="<%= button_type %>" method="<%= link_method %>">
```

Comment:
I'm not positive the disabled will work as we wanted. Can you confirm this button is able to be un-disabled? 
I believe the disabled attribute works a bit different than most - the presence of the key disables it, even if it is set to false. i.e. having 'disabled=false' will still show the button disabled. Possibly making this line 
`<%= "disabled" if disable_link == true %>` ?

Reply:
You're right, fixing this now.